app/controllers/api/v1/ping_thing_stats_controller.rb

```diff
@@ -12,11 +11,23 @@ def index
           interval: stats_params[:interval].to_i
         ).last(RECORDS_COUNT)
 
-        render json: create_json_result(stats), status: :ok
+        json_result = create_json_result(stats)
+        
+        respond_to do |format|
+          format.json { render json: json_result, status: :ok }
+          format.csv do
+            send_data convert_to_csv(index_csv_headers, json_result),
```

Comment:
Can you confirm that PingThingStat csv will be appropriately rendered in the production env then? I wasn't able to test it locally because my database didn't have PingThingStat records.

Reply:
It turned out the `as_json` method was missing. It is fixed.